docker/test/util/Dockerfile

```diff
@@ -19,25 +19,24 @@ RUN apt-get update \
         gnupg \
         iputils-ping \
         lsb-release \
+        xxd \
```

Comment:
It looks like this is redundant now?

Reply:
No. https://s3.amazonaws.com/clickhouse-test-reports/74201/106f627023699830d4dae270e812b5ce18551d95/fast_test.html

Reply:
What do I see here?

The xxd was necessary while it was used in the tests. But you replaced it with `{CLICKHOUSE_LOCAL}`. So package should be removed before merge

Reply:
Hm ok. It is no longer needed.